### PR TITLE
Adding question to FAQ.

### LINKED ISSUE #1032: Add FAQ entry: "Does SimpleITK offers all the functionalities as the C++ ITK?

### PR DESCRIPTION
**Pull request by zivy**
Adding a short answer on the difference between ITK and SimpleITK. Closes #1032

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -9,6 +9,18 @@ answers.
         :local:
         :backlinks: none
 
+General
+=======
+
+Does SimpleITK offer all the functionality as the C++ ITK?
+----------------------------------------------------------
+
+SimpleITK supports most ITK filters (:ref:`see list <lbl_filter>`) and the
```
Comment (blowekamp) [inline docs/source/faq.rst:18]: "most ITK filters" -> "most ITK image filters" and plug our image IO too.

Comment (zivy) [reply docs/source/faq.rst]: Done.

**Comment by zivy** [comment]
Not sure what the ghost flow is complaining about, there's nothing there.

**Comment by blowekamp** [comment]
> docs/source/faq.rst:13: leftover conflict marker

I'm guessing the "=====" mark up is being confused with merge conflict markup. Agreed, we can ignore it.

**Pull request merged**